player/cPlayerNotification.cpp

```diff
@@ -11,11 +12,11 @@ const char* eNotificationTypeString(const eNotificationType &type) {
             assert(false && "Unknown eNotificationType?");
             break;
     }
-    return "";
+    return {};
 }
 
 cPlayerNotification::cPlayerNotification(std::string msg, eNotificationType type) {
-    this->msg = msg;
+    this->msg = std::move(msg);
```

Comment:
aha, this does seem to be something you have to be aware of if you instantiate a `cPlayerNotification`? Or... wait, it probably copies `msg` already from the constructor args, then you simply move from there, thats what you intent correct?

Reply:
I started typing an explanation, but it's quite involved. Perhaps a better fix is to change the constructor of `cPlayerNotification` to take a string by const-ref. Because what was there copied the string twice.